### PR TITLE
Deployment examples

### PR DESCRIPTION
**Pull request by raveling**

### CODE REVIEW

File: docs/components/docs/GitHubExamplesCTA.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ export function GitHubExamplesCTA() {
         target="_blank"
         rel="noopener noreferrer"
       >
-        GitHub Repo <ArrowR />
+        Keystone GitHub Repo <ArrowR />
```
Comment (Noviny) [inline docs/components/docs/GitHubExamplesCTA.tsx:28]: Sentence case? (this is in part me learning our content guidelines)

**Review by bladey — APPROVED** [review]

File: docs/pages/docs/examples.tsx [hunk 2]
```diff
@@ -23,20 +29,221 @@ export default function Docs() {
...
+      <Type as="h2" look="heading30" margin="2rem 0 1rem 0" id="base-projects">
+        Base Projects
       </Type>
 
       <Type as="p" look="body18" margin="1.25rem 0 1.5rem 0">
-        Every project comes with its own docs that explain the how and why. They're useful as a
-        reference for best practice, and as a jumping off point when adding features to your own
-        Keystone project.
+        Simple starters to get you up and running with a local Keystone instance and understand the
+        basics of designing{' '}
```
Comment (Noviny) [inline docs/pages/docs/examples.tsx:46]: Should this be `basics of designing a schema` ? (missing 'a')

**Review by Noviny — APPROVED** [review]
Had two very small notes, but looks good!

File: docs/components/docs/GitHubExamplesCTA.tsx (hunk 1 above)
Comment (raveling) [reply docs/components/docs/GitHubExamplesCTA.tsx:28]: Fixed in #6528

File: docs/pages/docs/examples.tsx (hunk 2 above)
Comment (raveling) [reply docs/pages/docs/examples.tsx:46]: Fixed in #6528

**Pull request merged**